src/app/pages/details/details.component.ts

```diff
@@ -41,33 +52,46 @@ export class DetailsComponent extends ThreePanel {
         });
 
     const iref = anchorEl.getAttribute('data-iref');
-    const proj = anchorEl.hasAttribute('data-proj')?
-        anchorEl.getAttribute('data-proj') : this.project;
+    const proj = anchorEl.hasAttribute('data-proj')
+      ? anchorEl.getAttribute('data-proj')
+      : this.project;
 
     if (iref) {
       // navigates to details texts component
       this.router.navigate(
-        [ 'search-results',
-          this.chosenTermText,
-          'occurrences',
-          'texts'
-        ],
-        { queryParams: mergeParams(
-          {
-            proj: proj,
-            iref: iref,
-          },
-          this.route.snapshot.queryParams,
-          ['ga_lang', 'ga_isid', 'lang', 'isid', 'gw', 'type', 'name', 'pos']
-        )}
+        ['search-results', this.chosenTermText, 'occurrences', 'texts'],
+        {
+          queryParams: mergeParams(
+            {
+              proj: proj,
+              iref: iref
+            },
+            this.route.snapshot.queryParams,
+            ['ga_lang', 'ga_isid', 'lang', 'isid', 'gw', 'type', 'name', 'pos']
+          )
+        }
       );
     }
   }
 
   public handleMetadataClick(e) {
     e.preventDefault();
-    if (e.target.tagName === "A" && e.target.hasAttribute('data-zoom')) {
+    if (e.target.tagName === 'A' && e.target.hasAttribute('data-zoom')) {
       this.setZoom(parseInt(e.target.getAttribute('data-zoom'), 10));
     }
   }
+
+  private scrollToSelected(): void {
+    // Being doubly sure the element should be rendered, might not need the timeout
+    setTimeout(() => {
```

Comment:
I'm not really sure about adding code just in case. Did you find it not working sometimes in practice?

Reply:
this also doesn't actually seem to be triggered in the gh pages deploy 🤔 Have I put it in the wrong place?
[This](https://oracc.github.io/oracc-search-front-end/search-results/%C5%A1arr%C5%ABtu/occurrences/texts?proj=neo&iref=Q003387.61&ga_lang=akk&ga_isid=x0021873&lang=akk&isid=akk.r007bf0&gw=kingship&type=forms&name=LUGAL-su&pos=N) is what I've checked with

Reply:
That link is broken for me, but [this](https://oracc.github.io/oracc-search-front-end/search-results/dannu/occurrences/texts?proj=neo&iref=Q003702.146&ga_lang=akk&ga_isid=x0010078&lang=akk&isid=akk.r0011fc.p.s000&gw=strong&type=period&name=Neo-Assyrian&pos=AJ) also doesn't work.